mkdocs.yml

```
@@ -159,7 +159,7 @@ extra:
       link: https://medium.com/k0sproject
       name: k0s on Medium
     - icon: fontawesome/brands/discourse
-      link: https://forums.k8slens.dev/
+      link: https://kubernetes.slack.com/archives/C07VAPJUECS
```

We probably want to change the logo from discourse to slack.

```suggestion
    - icon: fontawesome/brands/slack
      link: https://kubernetes.slack.com/archives/C07VAPJUECS
```